src/Network/PacketStructure.js

```diff
@@ -184,6 +185,16 @@ define(['Utils/BinaryWriter', './PacketVerManager'], function(BinaryWriter, PACK
 	};
 
 
+	// 0x844
+	PACKET.CZ.HBT = function PACKET_CZ_HBT() {};
```

Comment:
Oh that, that actually meant for something in my friends server, its a security packet, I got few variable there that need to be removed including that one.